src/mahoji/commands/sell.ts

```diff
@@ -85,6 +86,30 @@ export const sellCommand: OSBMahojiCommand = {
 		});
 		if (bankToSell.length === 0) return 'No items provided.';
 
+		const castleWarsBank = new Bank();
+		const castleWarsLoot = new Bank();
+		for (const castleWarsBuyable of castleWarsBuyables) {
+			if (bankToSell.has(castleWarsBuyable.name)) {
+				castleWarsBank.add(castleWarsBuyable.name, bankToSell.amount(castleWarsBuyable.name));
+				castleWarsLoot.add(castleWarsBuyable.itemCost).multiply(bankToSell.amount(castleWarsBuyable.name));
+			}
+		}
+
+		if (castleWarsBank.length > 0) {
+			await handleMahojiConfirmation(
+				interaction,
+				`${user}, please confirm you want to sell ${castleWarsBank} for **${castleWarsLoot}**.`
+			);
+
+			await user.removeItemsFromBank(castleWarsBank);
+			await transactItems({
+				userID: user.id,
+				collectionLog: false,
+				itemsToAdd: castleWarsLoot
+			});
```

Comment:
transactItems can remove and add items at the same time. no need to do this separately